main/src/main/scala/sbt/Cross.scala

```diff
@@ -195,11 +195,11 @@ object Cross {
         commandsByVersion.flatMap {
           case (v, commands) =>
             commands match {
-              case Seq(c) => Seq(s"$SwitchCommand $verbose $v! $c")
+              case Seq(c) => Seq(s"$SwitchCommand $verbose $v $c")
               case Seq()  => Nil // should be unreachable
               case multi if fullArgs.isEmpty =>
                 Seq(s"$SwitchCommand $verbose $v! all ${multi.mkString(" ")}")
-              case multi => Seq(s"$SwitchCommand $verbose $v!") ++ multi
+              case multi => Seq(s"$SwitchCommand $verbose $v") ++ multi
```

Comment:
Hmm, in https://github.com/sbt/sbt/pull/5494 I didn't really know what I was doing and I aggressively removed the use of `!`. Is it necessary to remove the use of `!` in all three code occurrences to fix the issue?

After I sent https://github.com/sbt/sbt/pull/5494 I started thinking I might introduce other regressions.

PS: I noticed you didn't edit line `201` which makes me assume you _indeed_ change the minimum occurrences required 😂

Reply:
The reproducer doesn't need all 3 changes, but it would be quite inconsistent to not change all of these occurrences, as they seems quite similar. I'm definitely not familiar enough with this code to predict all the possible effects of this change, but at least the test suite doesn't appear to signal any regressions.